src/keystore.c

```diff
@@ -44,16 +44,16 @@ static bool _is_unlocked_bip39 = false;
 static uint8_t _retained_bip39_seed[64] = {0};
 
 #ifdef TESTING
-void mock_state(const uint8_t* retained_seed, const uint8_t* retained_bip39_seed)
+void mock_state(const uint8_t* seed, size_t seed_len, const uint8_t* bip39_seed)
 {
-    _is_unlocked_device = retained_seed != NULL;
-    if (retained_seed != NULL) {
-        _seed_length = KEYSTORE_MAX_SEED_LENGTH;
-        memcpy(_retained_seed, retained_seed, sizeof(_retained_seed));
+    _is_unlocked_device = seed != NULL;
+    if (seed != NULL) {
+        _seed_length = seed_len;
```

Comment:
i guess i missed it in a previous review but the doc comment for `_seed_length` says:

```c
// seed length defaults to 32 bytes, but we could accept smaller seeds in the future.
static uint8_t _seed_length = KEYSTORE_MAX_SEED_LENGTH;
```

the future is now :)

Reply:
Done - updated the comment and defaulting to 0 because why put any other meaningless value there.